### ISSUE TITLE
Commandline utilities are missing

### ISSUE THREAD

**Issue by nicorikken** [kyrofa/darktable-snap#13]
Darktable provides a number of commandline utilities which are currently missing from this snap:

- darktable-cli binary
- darktable-generate-cache binary
- darktable-chart binary
- darktable-cltest binary
- darktable-cmstest binary

More details: https://www.darktable.org/usermanual/en/overview_chapter.html#program_invocation

**Comment by jangop** [comment]
Any chance to get `darktable-cli` from this snap package in the foreseeable future?

**Comment by kyrofa** [comment]
Yes, this just keeps falling off my radar, I'm sorry. I'll try to get to it soon, although PRs are welcome in case anyone wants it sooner.

**Comment by jangop** [comment]
I would love to commit a PR, but, while `snapcraft.yaml` looks reasonably straightforward, I lack the necessary environment to test any changes I might make. Guess I'll have to wait; your efforts are much appreciated @kyrofa. 🙇
